packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -30,26 +29,25 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
```

Comment:
the obvious solution to me is listening for resize events. i remember this didn't work for some reason but i can look into it again.